lua/entities/cfc_super_cinderblock_projectile/shared.lua

```diff
@@ -35,13 +35,23 @@ function ENT:HitEffects( _, _, speed )
 end
 
 local vec_up = Vector( 0, 0, 1 )
+local criticalDamage = 100
 
 function ENT:PostHitEnt( hitEnt, damageDealt )
-    local critical = damageDealt >= 100
 
     util.ScreenShake( self:WorldSpaceCenter(), 5 + ( damageDealt * 0.5 ), 20, 0.5, 500 + damageDealt * 2 )
     util.ScreenShake( self:WorldSpaceCenter(), 5, 20, 0.1, 1500 + damageDealt )
 
+    if not ( hitEnt:IsPlayer() or hitEnt:IsNPC() ) then return end
+    if damageDealt >= criticalDamage then
+        hitEnt:EmitSound( "Flesh.ImpactHard", 90, 70, 1, CHAN_STATIC )
+        hitEnt:EmitSound( "Breakable.MatFlesh", 90, 70, 1, CHAN_STATIC )
+        hitEnt:EmitSound( "player/pl_fallpain1.wav", 95, 80, 1, CHAN_STATIC )
+        hitEnt:EmitSound( "npc/antlion/shell_impact4.wav", 95, math.random( 20, 30 ), 1, CHAN_STATIC )
+
+    end
+end
+function ENT:PostHit( hitEnt, _pos, _normal, _speed, damageDealt )
```

Comment:
Style + function signature change for `actuallyDidDamage` (still need to edit the function to use it of course, and the same for `PostHitEnt`)
```suggestion
    end
end

function ENT:PostHit( hitEnt, _pos, _normal, _speed, damageDealt, actuallyDidDamage )
```

The body of `PostHit` here also has similar styling problems, namely with extra and missing newlines.